packages/server/lib/server-base.ts

```diff
@@ -141,6 +158,73 @@ export class ServerBase<TSocket extends SocketE2E | SocketCt> {
     return this.ensureProp(this._httpsProxy, 'open')
   }
 
+  createServer (
+    app: Express,
+    config: Record<string, any>,
+    project: ProjectE2E | ProjectCt,
+    request: unknown,
+    onWarning: unknown,
+  ): Bluebird<[number, WarningErr?]> {
+    throw Error('`createServer must be implemented by super class!`')
+  }
```

Comment:
Hopefully one day we can have a single shared `createServer` function, and not re-implement it in two places, but that's not the case right now.

Reply:
Could we use an abstract class and make this method abstract?

Reply:
Done.